test_data/R/explore_dictionary.R

```diff
@@ -0,0 +1,47 @@
+library(tidyverse)
+library(lubridate)
+library(janitor)
+
+metadata <- read_csv("fr-dict.csv") %>% clean_names() %>% rename(text_validation_type = text_validation_type_or_show_slider_number)
+
```

Comment:
`fr-dict.csv` is not in the folder nor am I seeing an option to download the data dictionary from https://redcap.ctsi.ufl.edu/redcap/redcap_v9.3.5/DataExport/index.php?pid=8218.